packages/lab/src/breadcrumbs/internal/BreadcrumbsCollapsed.tsx

```diff
@@ -34,7 +37,7 @@ export const BreadcrumbsCollapsed: FC<BreadcrumbsCollapsedProps> = ({
   const { ref, shouldFocusOnMount } =
     useFocusMenuRemount<HTMLButtonElement>(key);
 
-  const itemToString = useCallback((child) => {
+  const itemToString = useCallback((child: ReactElement) => {
```

Comment:
Maybe could be Component<BreadcrumbProps> for better type safety (although obviously there are further TS issues with itemToString within CascadingMenu etc)

Reply:
Yeah, itemtoString is pretty messy to type. I can't remember why I chose this type initially.